qiskit_nature/algorithms/ground_state_solvers/minimum_eigensolver_factories/vqe_ucc_factory.py

```diff
@@ -33,23 +38,48 @@
 from ...initial_points import InitialPoint, HFInitialPoint
 from .minimum_eigensolver_factory import MinimumEigensolverFactory
 
+logger = logging.getLogger(__name__)
 
-class VQEUCCFactory(MinimumEigensolverFactory):
-    """A factory to construct a VQE minimum eigensolver with UCCSD ansatz wavefunction."""
 
+class VQEUCCFactory(MinimumEigensolverFactory):
+    """A factory to construct a VQE minimum eigensolver with UCCSD ansatz wavefunction.
+
+    Note: get_solver will overwrite any value that we directly set onto the vqe for both
+    the ansatz and the initial point. For example:
+    .. code-block:: python
+
+        factory = VQEUCCFactory()
+        factory.minimum_eigensolver.ansatz = UCCD()
```

Comment:
```suggestion
        factory.minimum_eigensolver.ansatz = UCC(excitations="d")
```
Same a in the vibrational case.